src/Messages/Interactions/TSInfoMessage.m

```diff
@@ -2,16 +2,40 @@
 //  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
 //
 
-#import "NSDate+millisecondTimeStamp.h"
 #import "TSInfoMessage.h"
+#import "NSDate+millisecondTimeStamp.h"
+#import <YapDatabase/YapDatabaseConnection.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
+NSUInteger TSInfoMessageSchemaVersion = 1;
```

Comment:
Just for discussion, it seems like there's no real reason to have separate classes for error vs. info message in the DB, right? It feels like the distinction only exists so as to print a red vs yellow border, which we could achieve much more simply. (not for this PR)